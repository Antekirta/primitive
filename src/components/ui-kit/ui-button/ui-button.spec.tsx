import UIButton from "./ui-button";
import React from "react";
import { create } from "react-test-renderer";

describe("ui-button component", () => {
  test("Matches the shapshot", () => {
    const uiButton = create(<UIButton></UIButton>);

    expect(uiButton.toJSON()).toMatchSnapshot();
  });
});
