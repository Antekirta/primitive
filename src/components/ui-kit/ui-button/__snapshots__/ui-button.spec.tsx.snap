// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui-button component Matches the shapshot 1`] = `
<button
  className="ui-button undefined"
/>
`;
